flexmeasures/cli/data_add.py

```diff
@@ -2007,13 +2007,18 @@ def create_asset_with_one_sensor(
         unit: str = "MW",
         **asset_attributes,
     ):
+        asset_kwargs = dict()
+        if asset_attributes.get("parent_asset_id"):
```

Comment:
Let's promote `parent_asset_id` to a real parameter of `create_asset_with_one_sensor`.

Reason: It's not really one of the asset attributes we handle, but an actual data model field.